### ISSUE TITLE
Example with roll time appears to be wrong.

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Queue#635]
The following text/code from `docs/timezone_rollover.adoc` appears to indicate a roll over time at 17 and not at midnight:

The following code extract is an instruction to use America/New_York as time zone, and rolls over queues at midnight regardless of offset to UTC.
```
.rollTime(LocalTime.of(17, 0), ZoneId.of("America/New_York"))
```

**Comment by RobAustin** [comment]
mmm -  maybe it needs to be made clearer but this feature is only available as an enterprise feature. - it mentions Enterprise Edition in the title of the doc, Timezone Queue Rollover (Enterprise Edition Feature)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.19.0](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.19.0), [BOM-2.19.4](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.4)
